src/api/app/views/webui/kiwi/images/_preferences.html.haml

```diff
@@ -1,20 +1,22 @@
 .nested-fields
   %h2#image-name
     Preferences
-  %ul.inline
-    %li
-      %strong Image type:
-      %span.fill{ data: { tag: 'type_image' } }= @image.preference.type_image
-    %li
-      %strong Version:
-      %span.fill{ data: { tag: 'version' } }= @image.preference.version
-    - if @image.preference.containerconfig_fields_editable?
+  - @image.preferences.each.with_index(1) do |preference, index|
+    %h4= "##{index}"
```

Comment:
Can't this be `preference.profiles`?

Reply:
Right now, it's not possible. To have this, we would need to change the Kiwi::Preference model to have an association with the Kiwi::Profile model. I didn't do it to limit the scope of the changes.

Reply:
I'm afraid otherwise this is not really useable. How do I know what this is about without prior knowledge of the XML structure?

Reply:
You are right.

So according to the Kiwi documentation, [a preference has one profile name](https://opensource.suse.com/kiwi/development/schema.html#id291). I will build the association accordingly.